### ISSUE TITLE
Question: Not finding Hummingbird board option in Arduino IDE board manager

### ISSUE THREAD

**Issue by Fireflaker** [BirdBrainTechnologies/DuoFirmwareBurner#1]
![image](https://user-images.githubusercontent.com/22672963/36633281-89077c3a-1960-11e8-88a6-1271dec8f851.png)
What should I do? Should I install it manually? 
Thanks

**Comment by birdbraintech** [comment]
Yes, make sure to follow the instructions here: http://www.hummingbirdkit.com/learning/setting-arduino-environment-hummingbird-duo

You can check that the Hummingbird library files are in your directory by going to Arduino Sketchbook folder->hardware (or libraries) and making sure that there's a hummingbirdduo folder in there.
